1-js/09-classes/02-class-inheritance/article.md

```diff
@@ -352,7 +352,7 @@ Why is there the difference?
 
 Well, the reason is in the field initialization order. The class field is initialized:
 - Before constructor for the base class (that doesn't extend anything),
-- Imediately after `super()` for the derived class.
+- Immediately after `super()` for the derived class.
```

Comment:
¿Por qué este artículo solo está traucido a partir de la línea 367?
¿Fue un error de sync o estaba así originalmente?

Reply:
@vplentinax 
no sé, pero PUDE vencer la manía , me dije FOCuS! 
puse el issue (vas a ver varios) y segui adelante, 

ahora podria verlo, en "blame" not to blame but to avoid the issue in the future....

Reply:
Ok